jerry-libm/trig.c

```diff
@@ -190,6 +184,11 @@ static const int ipio2[] =
 static int
 __kernel_rem_pio2 (double *x, double *y, int e0, int nx, int prec)
 {
+  const double zero   = 0.0;
+  const double one    = 1.0;
```

Comment:
Floating point representation changed (to an equivalent one, but changed), and comment on binary representation got removed (both for zero and one).